linker error for bullet soft body

When compiling the `gazebo_ros_pkgs` on mojave, I run into the following linker errors:

```
[14%] Linking CXX shared library libgazebo_ros_node.dylib
[ 17%] Linking CXX shared library libgazebo_ros_utils.dylib
ld: library not found for -lBulletSoftBody
clang: error: linker command failed with exit code 1 (use -v to see invocation)
ld: library not found for -lBulletSoftBody
make[2]: *** [libgazebo_ros_node.dylib] Error 1
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[1]: *** [CMakeFiles/gazebo_ros_node.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
make[2]: *** [libgazebo_ros_utils.dylib] Error 1
make[1]: *** [CMakeFiles/gazebo_ros_utils.dir/all] Error 2
make: *** [all] Error 2
--- stderr: gazebo_ros
ld: library not found for -lBulletSoftBody
clang: error: linker command failed with exit code 1 (use -v to see invocation)
ld: library not found for -lBulletSoftBody
make[2]: *** [libgazebo_ros_node.dylib] Error 1
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[1]: *** [CMakeFiles/gazebo_ros_node.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
make[2]: *** [libgazebo_ros_utils.dylib] Error 1
make[1]: *** [CMakeFiles/gazebo_ros_utils.dir/all] Error 2
make: *** [all] Error 2
---
Failed   <<< gazebo_ros	[ Exited with code 2 ]
```

I am a bit puzzled about it because just before, it correctly finds the bullet dependencies:
```
-- Checking for module 'bullet>=2.82'
--   Found bullet, version 2.88
```
And indeed, it seems like bullet is correctly installed:
```
 ➭ locate libBulletSoftBody
/usr/local/Cellar/bullet/2.88/lib/libBulletSoftBody.2.88.dylib
/usr/local/Cellar/bullet/2.88/lib/libBulletSoftBody.a
/usr/local/Cellar/bullet/2.88/lib/libBulletSoftBody.dylib
/usr/local/lib/libBulletSoftBody.2.88.dylib
/usr/local/lib/libBulletSoftBody.a
/usr/local/lib/libBulletSoftBody.dylib
```

any idea on what's going wrong here?

--- a/gazebo_ros/CMakeLists.txt
+++ b/gazebo_ros/CMakeLists.txt
@@ -25,6 +25,9 @@ include_directories(
   include
   ${gazebo_dev_INCLUDE_DIRS}
 )
+link_directories(
+  ${gazebo_dev_LIBRARY_DIRS}
+)
 
 # gazebo_ros_node
 add_library(gazebo_ros_node SHARED

